examples/cms-makeswift/.env

```diff
@@ -0,0 +1,2 @@
+MAKESWIFT_API_HOST=https://api.makeswift.com
```

Comment:
Let's rename this file to `.env.local.example` to match other examples and be sure that API_KEY's won't be accidentally committed.